apidoc/Titanium/Yahoo/Yahoo.yml

```diff
@@ -14,6 +14,10 @@ methods:
       - name: callback
         summary: the function to execute when the query completes.
         type: Callback<YQLResponse>
+deprecated:
+    since: "8.0.0"
```

Comment:
didn't we deprecate this before?

Reply:
I think we planned to in 7 but never came to it